### PR TITLE
fix: use filepath when handling OS paths

### PR DESCRIPTION
**Pull request by Stebalien**

### CODE REVIEW

File: add.go [hunk 1]
```diff
@@ -4,9 +4,9 @@ import (
 	"context"
 	"encoding/json"
 	"errors"
+	"filepath"
```
Comment (djdv) [inline add.go]:
```suggestion
	"path/filepath"
```

Comment (Stebalien) [reply add.go]: (oops)

**Comment by djdv** [comment]
@Stebalien 
My Github suggestion was actually invalid because it only changed the line contents, `go fmt` still wants that imports ordered a specific way. (oops)

**Comment by Stebalien** [comment]
(fixed...)

**Pull request merged**